packages/type/scss/_reset.scss

```diff
@@ -75,7 +75,7 @@
   }
 
   a {
-    color: #0062ff;
+    color: $interactive-01;
```

Comment:
If I remember correctly, I _think_ it's a hex value due to this token coming from the themes package. Since themes depends on type, if we had type depends on themes then we would have a cycle between the two.

Reply:
then should we remove this hard hex from the reset?

Reply:
@DusanMilko maybe we could swing something like:

```scss
  a {
    @if global-variable-exists($carbon--theme) and
      map-has-key($carbon--theme, 'link-01')
    {
      color: map-get($carbon--theme, 'link-01');
    } @else {
      color: #0062ff;
    }
  }
```

?

Reply:
Do we even need to reset the color? It doesn't look like we reset any other color values?

Reply:
@tw15egan I think you're right that we don't need to reset colors, I assume that when the default type mixin was made it was just translating over from the v9 default styles which we could totally just drop in v11 👍